### PR TITLE
Scavs: Add a command to forcibly queue hunts 

### PR DESCRIPTION
**Pull request by Lucas-Meijer**
Add the ability to forcequeue hunts, which disregard empty answers and short hunts.

As requested by Room Owner Computerwizard8800

### CODE REVIEW

**Review by PartMan7 — APPROVED** [review]
Hi, looks good (maybe look into twists, but that probably won't need to be done since no one in their right mind would forcequeue a twist with missing answers, hopefully)

Outside the scope of this PR, the number of combos for commands is starting to get a bit unwieldy; we should probably look into a way to autogenerate these variants

**Review by Zarel — APPROVED** [review]

File: server/chat-plugins/scavengers.ts [hunk 1]
```diff
@@ -978,23 +978,26 @@ export class ScavengerHunt extends Rooms.RoomGame<ScavengerHuntPlayer> {
...
 			if (i % 2) {
 				const answers = question.split(';').map(p => p.trim());
 				formattedQuestions[i] = answers;
-				if (!answers.length || answers.some(a => !toID(a))) {
+				if (!force && (!answers.length || answers.some(a => !toID(a)))) {
 					return {err: "Empty answer - only alphanumeric characters will count in answers."};
 				}
 			} else {
+				// Skip last question if there is no answer
+				if (i + 1 === questionArray.length) { continue; }
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:996]:
```suggestion
				if (i + 1 === questionArray.length) {
								continue;
				}
```

Comment (Zarel) [reply server/chat-plugins/scavengers.ts:996]: Why not just
```suggestion
				if (i + 1 === questionArray.length) continue;
```

**Comment by Lucas-Meijer** [comment]
@mia-pi-git @Zarel I've updated that part, thanks!

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**